src/scripts/main.js

```diff
@@ -1 +1,82 @@
 'use strict';
+'use strict';
```

Comment:
Don't need to duplicate 'use strict'
```suggestion
'use strict';
```